__tests__/app.test.js

```diff
@@ -69,16 +69,16 @@ describe('GET /api/articles/:article_id', () => {
       .expect(200)
       .then(({body}) => {
           const {article} = body;
-          expect(article).toEqual({
+          expect(article).toMatchObject({
               article_id: 1,
-              title: "Living in the shadow of a great man",
-              topic: "mitch",
-              author: "butter_bridge",
-              body: "I find this existence challenging",
-              created_at: "2020-07-09T20:11:00.000Z",
-              votes: 100,
-              article_img_url:
-                "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              created_at: expect.any(String),
+              comment_count: expect.any(String),
```

Comment:
for this test you should probably be more specific here with the actual number of comments